native-plasma/app/src/main/jni/plasma.c

```diff
@@ -1,503 +0,0 @@
-/*
```

Comment:
I have to admit I didn't, bu there is very few lines in common between plasma.c and plasma.cpp since I move the rendering part out of the plasma.cpp.